css/style.css

```diff
@@ -37,23 +37,188 @@ Developer @GA in NYC
   float: left;
   padding: 0 10px 10px 0; /* note the padding around a left floated image */
 }
+.categories {
+  display: block;
+  font-weight:300;
+  font-size:24px;
+}
+.nav li {
+  display: inline-block;
+  color: #f9e42e;
+  font-weight: 600;
+  font-size: 32px;
+  padding: 8px;
+}
+.nav ul {
+  list-style: none;
+  overflow: auto;
+  width: 600px;
+  margin: 0 auto;
+
+}
+.signup {
+  float: left;
+  width: 100%;
+  background: #f9e42e;
+  margin: auto;
+  padding: 50px;
+}
+
+.SignUpB {
+  background-color: #033048;
+  font-weight: 800;
+  font-size: 24px;
+  border-radius: 5px;
+  padding: 10px 35px;
+  margin: 5px auto;
+  display: block;
+  border: none;
+  color: white;
+}
+.social{ 
+  margin: auto;
+  text-align: center;
+}
+.read{
+  float: right;
+  width: 25%;
+  text-decoration: none;
+  font-size: 22px;
+  font-weight: 600;
+  color: #033048;
+  padding: 15px;
+}
+.about {
+  padding: 20px;
+}
 
 /******************************************
 /* BASE STYLES
 /*******************************************/
 
-body {
+main {
+  float: left;
+  width: 700px;
+  padding: 20px;
+}
+body{
+  font-family: 'Open Sans', Sans-Serif; 
+  background-color: #f0efef;
+}
+header {
+  background-color: #033048;
+  min-height: 100px;
+  padding: 20px;
+}
+nav {
+  font-weight: 800;
+  padding: 20px;
+}
+ul {
+  display: inline;
+}
+li {
+  display: inline;
+}
+
+h1 {
+  float: left;
+  width: 30%;
+  font-family: 'Open Sans', Sans-Serif; 
+  color: white;
+  font-size: 40px;
+  font-weight: 600;
+  padding: 20px;
+}
+h2 {
+  font-weight: 600;
+  font-size: 40px;
+  font-family: 'Open Sans', Sans-Serif;
+}
+h3 {
+  font-weight: 800;
+  font-size: 32px;
+  padding: 10px;
+}
+h6 {
+  font-weight: 300;
+}
+p {
+  font-weight: 300;
+  font-size: 24px; 
+  color: #606161
+}
+p.text:first-letter {
+  float: left;
+  font-size: 75px;
+  line-height: 60px;
+  padding-top: 4px;
+  padding-right: 8px; 
+  padding-left: 3px;
+}
+button {
+  font-weight: 800;
+  font-size: 30px;
+  background-color: #033048;
+  border-radius: 5px;
+  padding: 10px 35px;
+  margin: 5px auto;
+  display: block;
+  border: none;
+  color: white;
+}
+
+aside {
+  float: left;
+  width: 240px;
+  padding: 20px;
```

Comment:
Since you have padding of `20px` on the `main` and you have the same here on `aside`, you're actually creating a `40px` gutter (the right padding of `main` + the left padding of `aside`). To adjust this, you can simply do something like this:

```
aside {
 padding: 20px; //sets 20px of padding all the way around the box
 padding-left: 0; //override the left padding to be 0
}
```